src/components/CloseChatButtons.tsx

```diff
@@ -6,22 +6,22 @@ import * as FlexWebChat from '@twilio/flex-webchat-ui';
 import Exit from './QuickExitButton';
 import End from './EndChatButton';
 
-const CloseChatButtons = ({ channelSid, token }: MapStateToProps) => {
-  if (!channelSid || !token) {
+const CloseChatButtons = ({ channelSid, token, language }: MapStateToProps) => {
```

Comment:
Do we want to not render the button or default to English if there's no language? Should we always have a language?